tooling/hcpctl/internal/e2e/e2e_test.go

```diff
@@ -679,3 +873,108 @@ func TestHCPList(t *testing.T) {
 		})
 	}
 }
+
+func TestReleaseStatus(t *testing.T) {
+	tests := []struct {
+		name         string
+		outputFormat string
+		releaseName  string
+		namespace    string
+	}{
+		{
+			name:         "release status in SC context",
+			outputFormat: "yaml",
+		},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			t.Parallel()
+			ctx, cancel := context.WithTimeout(context.Background(), DefaultTimeout)
+			defer cancel()
+
+			// Create a temporary kubeconfig for this test
+			kubeconfigPath := createTempKubeconfig(t, "sc-release-status")
+
+			// Get SC access via breakglass
+			_, err := execCommand(ctx, HCPCTLBinary, "sc", "breakglass", DefaultSCCluster, "--no-shell", "--output", kubeconfigPath)
+			if err != nil {
+				t.Fatalf("sc breakglass failed: %v", err)
+			}
+
+			// Verify kubeconfig was created
+			if _, err := os.Stat(kubeconfigPath); os.IsNotExist(err) {
+				t.Fatal("kubeconfig file was not created")
+			}
+
+			// Build release status command args
+			args := []string{"release", "status", "--kubeconfig", kubeconfigPath, "-o", tt.outputFormat}
```

Comment:
Unless you _really_ need/want to test the flag binding logic, you can create raw options here and run the rest without `exec`, which gives you test coverage for everything downstream and doesn't incur the cost of exec.